synapse/lib/storm.py

```diff
@@ -1331,11 +1332,6 @@ def _stormOperSetProp(self, query, oper):
             if props:
                 [core.setTufoProps(node, **props) for node in nodes]
 
-    def _iterPropTags(self, props, tags):
```

Comment:
This was unused

Reply:
can't find it anywhere else in the codebase

Reply:
was support code for _stormOperAllTag, no longer used (see 5ecbf19f14fbc8fe8a53718130d139a286e4ad10)